lib/msal-common/src/utils/Constants.ts

```diff
@@ -202,22 +146,6 @@ export const PromptValue = {
     NO_SESSION: "no_session",
 };
 
-/**
- * SSO Types - generated to populate hints
- */
-export const SSOTypes = {
-    ACCOUNT: "account",
-    SID: "sid",
-    LOGIN_HINT: "login_hint",
-    ID_TOKEN: "id_token",
-    DOMAIN_HINT: "domain_hint",
-    ORGANIZATIONS: "organizations",
-    CONSUMERS: "consumers",
-    ACCOUNT_ID: "accountIdentifier",
-    HOMEACCOUNT_ID: "homeAccountIdentifier",
```

Comment:
Some of these removed don't appear in the new AADServerParamKeys. Were they not needed/used at all?

Reply:
They were not referenced anywhere, likely weren't cleaned up when some feature or another was changed or removed